addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -100,6 +118,52 @@ odoo.define('website_slides.quiz', function (require) {
             this.$('.o_wslides_js_lesson_quiz_error').removeClass('d-none');
         },
 
+        /**
+         * Allows to reorder the questions
+         * @private
+         */
+        _bindSortable: function() {
+            this.$el.sortable({
+                handle: '.o_wslides_js_quiz_sequence_handler',
+                items: '.o_wslides_js_lesson_quiz_question',
+                stop: this._reorderQuestions.bind(this),
+                placeholder: 'o_wslides_js_quiz_sequence_highlight position-relative my-3'
+            });
+        },
+
+        /**
+         * Get all the questions ID from the displayed Quiz
+         * @returns {Array}
+         * @private
+         */
+        _getQuestionsIds: function () {
+            var questionIds = [];
+            this.$('.o_wslides_js_lesson_quiz_question').each(function () {
+                questionIds.push($(this).data('question-id'));
+            });
+            return questionIds;
+        },
+
+        _modifyQuestionsSequence: function () {
+            this.$('.o_wslides_js_lesson_quiz_question').each(function (index, question) {
+                $(question).find('span.o_wslides_quiz_question_sequence').text(index + 1);
+            });
+        },
+
+        /**
+         * RPC call to resequence all the questions
+         * @private
+         */
+        _reorderQuestions: function () {
+            this._rpc({
+                route: '/web/dataset/resequence',
+                params: {
+                    model: "slide.question",
+                    ids: this._getQuestionsIds()
+                }
+            }).then(this._modifyQuestionsSequence())
```

Comment:
Does this actually work?

I would normally say `.then(this._modifyQuestionsSequence.bind(this));`

Reply:
It does work :)
I read that the bind method create a new function with a new context passed in parameter. but the context here is the same.

I applied the change anyway !